static/src/js/components/Timeline/__tests__/Timeline.test.js

```diff
@@ -155,24 +155,59 @@ describe('Timeline component', () => {
     }])
   })
 
+  test('timeline displays on focused collection even when projectCollectionsIds is empty', () => {
+    const { enzymeWrapper } = setup({
+      pathname: '/search/granules', // Indicating it's not a project page
+      collectionMetadata: {
+        someCollection: {
+          title: 'Some Collection'
+        }
+      },
+      projectCollectionsIds: [], // Empty project collections
```

Comment:
If you add a collection to your project and then go back to the focused collection timeline for a different collection isn't it going to execute both workflows?

Reply:
Right, I suppose you don't want to see both timelines in that view, correct?

Reply:
Fixed